tic-tac-toe/src/app/components/App/index.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component, Fragment } from 'react';
+import { Route, Redirect } from 'react-router';
+import { ConnectedRouter } from 'connected-react-router';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { history } from '../../../redux/store';
+import '../../../scss/index.scss';
+import Game from '../../screens/Game';
+import Login from '../../screens/Login';
+import actionLogin from '../../../redux/Login/actions';
+
+class App extends Component {
+  componentDidMount() {
+    this.props.dispatch(actionLogin.token(localStorage.getItem('token')));
+  }
+  render() {
+    const { tokenLoading } = this.props;
+    return (
+      <ConnectedRouter history={history}>
+        <Fragment>
+          <Route exact path="/" component={Login} />
+          <Route path="/game" component={Game} />
+          {tokenLoading ? (
+            <div />
+          ) : (
+            <div>
+              <Route
+                render={() => (this.props.userToke.length ? <Redirect to="/game" /> : <Redirect to="/" />)}
```

Comment:
Add path='/game' here and remove the another Route

Reply:
I can't delete this <Redirect to="/" />  Because this is necessary for validation when does not have token.